### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ndought**
Login page- Still need to pick color schemes so these colors are just to display the animations.

### CODE REVIEW

**Review by JHardeman — APPROVED** [review]

File: frontend/login.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              <div id="nav-expanded" class="nav-items">
+                <div class="nav-brand">
+        
+                </div>
+                <div>Some Stuff</div>
+                <div>Some Stuff</div>
+                <div>Some Stuff</div>
+              </div>
+            </div>
+          </nav>
```
Comment (ColinWake) [inline frontend/login.html:40]: Do we need a hamburger button for the login page?

File: frontend/login.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+        <form action="/login" class="box" method="POST">
+          <h1>Login</h1>
+          <input type="text" name="" placeholder="Username">
+          <input type="password" name="" placeholder="Password">
+          <input type="submit" name="" value="Login"
```
Comment (ColinWake) [inline frontend/login.html]:
```suggestion
          <input type="text" name="username" placeholder="Username">
          <input type="password" name="password" placeholder="Password">
          <input type="password" name="retypePassword" placeholder="Retype Password">
          <button type="submit">Login</button>  
```
We have a `retypePassword` field in our User POJO, and the `name` attribute is what is sent to the backend as the key of the parameter, so we need that too.

**Review by ColinWake — CHANGES_REQUESTED** [review]

File: frontend/login.html (hunk 1 above)
Comment (NightingaleMedia) [reply frontend/login.html:40]: @ColinWake  keep it for now will be easy to delete later.  might want to maintain basic app functionality even if user is not logged in.

File: frontend/login.html (hunk 2 above)
Comment (NightingaleMedia) [reply frontend/login.html]: ^^ agree with this tho.

**Review by caleb-bb — APPROVED** [review]

**Review by ColinWake — APPROVED** [review]

File: frontend/login.html (hunk 2 above)
Comment (ColinWake) [reply frontend/login.html]: Resolved in 2fcf106

**Review by NightingaleMedia — APPROVED** [review]

**Pull request merged**